web/src/lib/elements/ui/link/link.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import { Button } from '../button';
+
+  export let text: string;
+  export let url: string;
+  let active: boolean = false;
+
+  onMount(() => {
+    active = window.location.pathname === url;
+  });
```

Comment:
You can get rid of this and simply use `$: active = $page.url === url`

Reply:
done